SwiftParseChat/ProfileTableViewController.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  ProfileTableViewController.swift
+//  SwiftParseChat
+//
+//  Created by Justin (Zihao) Zhang on 4/26/15.
+//  Copyright (c) 2015 Jesse Hu. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+protocol EditProfileDelegate {
+    func didSelectProfileTableRow(segueID: String, action: String)
+}
+
+class ProfileTableViewController: UITableViewController {
+    
+    let actionItemsWithoutFB = [EDIT_PROFILE_NAME, EDIT_EMAIL, EDIT_PASSWORD, SEND_FEEDBACK]
+    let actionItemsWithFB = [EDIT_PROFILE_NAME, EDIT_EMAIL, SEND_FEEDBACK]
+    var delegate: EditProfileDelegate!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+    
+    override func viewDidAppear(animated: Bool) {
+        super.viewDidAppear(animated)
+        self.tableView.reloadData()
+    }
+    
+    override func numberOfSectionsInTableView(tableView: UITableView) -> Int {
+        return 1
+    }
+    
+    override func tableView(tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        if isLoginByFB() {
+            return actionItemsWithFB.count
+        } else {
+            return actionItemsWithoutFB.count
+        }
+    }
+    
+    override func tableView(tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
+        return 0.01
```

Comment:
If put it as 0.01, then you will see many empty lines. By setting 0.01, we have a nice border